arviz/plots/autocorrplot.py

```diff
@@ -30,19 +30,19 @@ def plot_autocorr(
     Parameters
     ----------
     data: obj
-        Any object that can be converted to an az.InferenceData object
-        Refer to documentation of az.convert_to_dataset for details
+        Any object that can be converted to an :class:`arviz.InferenceData` object
+        Refer to documentation of :func:`arviz.convert_to_dataset` for details
     var_names: list of variable names, optional
-        Variables to be plotted, if None all variable are plotted. Prefix the
-        variables by `~` when you want to exclude them from the plot. Vector-value
+        Variables to be plotted, if None all variables are plotted. Prefix the
+        variables by ``~`` when you want to exclude them from the plot. Vector-value
         stochastics are handled automatically.
     filter_vars: {None, "like", "regex"}, optional, default=None
         If `None` (default), interpret var_names as the real variables names. If "like",
         interpret var_names as substrings of the real variables names. If "regex",
-        interpret var_names as regular expressions on the real variables names. A la
-        `pandas.filter`.
+        interpret var_names as regular expressions on the real variables names. A
+        ``pandas.filter``.
```

Comment:
that was French meaning "like", now it has a different meaning, not completely sure which